terraform/builder-monolith.tf

```diff
@@ -20,6 +20,21 @@ resource "aws_instance" "monolith" {
         agent       = "${var.connection_agent}"
     }
 
+    ebs_block_device {
```

Comment:
@jtimberman I think you're right, there doesn't seem to be a nice way to express what we want with Terraform right now so we should just go ahead and make the EBS part of the lifecycle of the instance